### PR TITLE
Added Convert_md2pdf.py script that,Create python script that convert

### PR DESCRIPTION
**Pull request by antongovorov**

### CODE REVIEW

**Comment by KhrystynaDubovyk** [comment]
@abyzhynar, please review

File: docs/Utils/Convert_md2pdf.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import os 
+import pandoc
+from markdown import markdown
+import pdfkit
+import re
+#Split all .md in one
+directory = '/home/agovorov/Desktop/RequirementsTest/docs/Guide_index/'
```
Comment (AByzhynar) [inline docs/Utils/Convert_md2pdf.py]: @KhrystynaDubovyk Not universal path, isn't it? Probably script should be changed to allow to setup this path as script parameter.
@antongovorov Looks that it should be changed by you. Am I wrong?

**Review by AByzhynar — CHANGES_REQUESTED** [review]

File: docs/Utils/Convert_md2pdf.py (hunk 1 above)
Comment (antongovorov) [reply docs/Utils/Convert_md2pdf.py]: Sorry, you are right

Comment (antongovorov) [reply docs/Utils/Convert_md2pdf.py]: fix 4126e66175086736027039d1278a14590241b7cb

Comment (AByzhynar) [reply docs/Utils/Convert_md2pdf.py]: @antongovorov Can we change it to use path as parameter or add information about the location from where this script should be run? Not so good idea to hardcode relative path. If we change location of Guide somehow afterwards - script will stop working.

**Comment by AByzhynar** [comment]
@antongovorov Can we change it to use path as parameter or add information about the location from where this script should be run? Not so good idea to hardcode relative path. If we change location of Guide somehow afterwards - script will stop working.

File: docs/Utils/Convert_md2pdf.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import os
```
Comment (malirod) [inline docs/Utils/Convert_md2pdf.py:1]: @antongovorov , please run this script through pylint and fix all warnings.

**Review by malirod — CHANGES_REQUESTED** [review]

**Comment by malirod** [comment]
@AByzhynar, we don't need new params for now.

**Review by AByzhynar — APPROVED** [review]

**Pull request merged**